app/onboarding/page.tsx

```diff
@@ -162,34 +227,48 @@ const ReviewPage = ({
   };
 
   return (
-    <PageContainer>
-      <ReviewContainer>
-        <H3 style={{ textAlign: 'center' }}>Review Your Response</H3>
-        <LabeledCustomSelect
-          label="State Location"
-          value={selectedState}
-          options={stateOptions}
-          onChange={setSelectedState}
-        />
+    <OnboardingContainer>
+      <div style={{ width: '100%', display: 'flex', flexDirection: 'column' }}>
+        <ProgressBar progress={100} />
+        <H3
+          $color={COLORS.shrub}
+          style={{
+            textAlign: 'center',
+            marginTop: '90px',
+            marginBottom: '40px',
+          }}
+        >
+          Review & Submit
+        </H3>
+        <ContentContainer>
+          <P1 style={{ color: COLORS.shrub, marginBottom: '16px' }}>
+            Your Responses
+          </P1>
+          <LabeledCustomSelect
+            label="State Location"
+            value={selectedState}
+            options={stateOptions}
+            onChange={setSelectedState}
+          />
 
-        <LabeledCustomSelect
-          label="Garden Type"
-          value={selectedGardenType}
-          options={gardenTypeOptions}
-          onChange={setSelectedGardenType} // Directly pass the selected value
-        />
-        <LabeledCustomSelect
-          label="Plot Status"
-          value={selectedPlot}
-          options={plotOptions}
-          onChange={value => setSelectedPlot(value === true)} // Convert the value as needed
-        />
-        <div style={{ height: '12.625rem' }} />
-        <BigButton color={COLORS.shrub} onClick={handleSubmit}>
-          Let&apos;s Start Growing !
-        </BigButton>
-      </ReviewContainer>
-    </PageContainer>
+          <LabeledCustomSelect
```

Comment:
note: instead of creating a new component, we want to just add a new prop to CustomSelect